client/src/common/components/Toolbar/components/UserBar/UserBar.scss

```diff
@@ -94,6 +94,7 @@
   }
 
   &__menu-logout {
+    font-family: inherit;
```

Comment:
why do you need this?? 
Did font was different from other elements on the app??

Reply:
Without this rule, the logout button has a different font like the whole app. It's strange because it should inherit font-family by default, but it doesn't.